vue/index.html

```diff
@@ -6,6 +6,6 @@
     </head>
     <body>
         <div id="app"></div>
-        <script src="dist/build.js"></script>
+        <script src="/js/build.js"></script>
```

Comment:
No I didn't as you asked me not to. You told me you would serve the build.js from static/js, with the other js files.

But that's no problem, I can add that.